pom.xml

```diff
@@ -665,3920 +650,525 @@
                 <version>${ee.maven.version}</version>
+                <type>zip</type>
             </dependency>
 
             <dependency>
                 <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-singleton-api</artifactId>
+                <artifactId>wildfly-standard-ee-bom</artifactId>
                 <version>${ee.maven.version}</version>
+                <type>pom</type>
             </dependency>
+
             <dependency>
                 <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-singleton-extension</artifactId>
+                <artifactId>wildfly-standard-expansion-bom</artifactId>
                 <version>${ee.maven.version}</version>
+                <type>pom</type>
             </dependency>
+
             <dependency>
                 <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-singleton-server</artifactId>
+                <artifactId>wildfly-testsuite-shared</artifactId>
                 <version>${ee.maven.version}</version>
+                <scope>test</scope>
             </dependency>
+
             <dependency>
-                <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-singleton-service</artifactId>
-                <version>${ee.maven.version}</version>
+                <groupId>${full.maven.groupId}</groupId>
+                <artifactId>wildfly-build</artifactId>
+                <version>${full.maven.version}</version>
+                <type>pom</type>
             </dependency>
 
             <dependency>
-                <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-web-api</artifactId>
-                <version>${ee.maven.version}</version>
+                <groupId>${full.maven.groupId}</groupId>
+                <artifactId>wildfly-dist</artifactId>
+                <version>${full.maven.version}</version>
+                <type>pom</type>
             </dependency>
+
             <dependency>
-                <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-web-cache</artifactId>
-                <version>${ee.maven.version}</version>
+                <groupId>${full.maven.groupId}</groupId>
+                <artifactId>wildfly-feature-pack-galleon-content</artifactId>
+                <version>${full.maven.version}</version>
+                <type>pom</type>
             </dependency>
+
             <dependency>
-                <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-clustering-web-container</artifactId>
-                <version>${ee.maven.version}</version>
+                <groupId>${full.maven.groupId}</groupId>
+                <artifactId>wildfly-feature-pack-galleon-content</artifactId>
+                <version>${full.maven.version}</version>
+                <type>zip/</type>
```

Comment:
There are some of these, notice additional / in `zip/`.

I've just checked these errors are also in main, which means these dependencies are not being used at all

Reply:
We should clean those up. But not in this PR. ;)

I don't think they are used. The use case for them is the way WF Core produces them and WF uses the zip. For these local ones we just directly use the local source though. If we every moved the FP build to a separate project they'd be useful.